cmd/overlay.go

```diff
@@ -19,16 +19,20 @@ Options:
 	--verbose/-v		enable verbose output
 	
 Commands:
-	new [directory]		overlay a directory
-	commit			commit the changes
-	discard			discard the changes
-	list			list the active overlays
+	new [directory]			Overlay a directory
+	commit [directory]		Commit the changes
+	discard [directory]		Discard the changes
+	list					List the active overlays
 
 Examples:
-	almost overlay new /etc/cute-path
-	almost overlay commit
-	almost overlay discard
-`)
+	# almost overlay new /etc/cute-path
+	# almost overlay commit /etc/cute-path
+	# almost overlay discard /etc/cute-path
+	# almost overlay list
+`
+
+func overlayUsage(*cobra.Command) error {
+	fmt.Print(USAGE_MSG)
```

Comment:
why is this a var now?

Reply:
This was an artifact of me  that I forgot to revert from commit 72a2cd38a1a325801aff518d238c43760490b0e9 in the switch statement. This is not needed anymore